### ISSUE TITLE
" watch " breaks brat

### ISSUE THREAD

**Issue by bholland** [nlplab/brat#1241]
This is really strange. I have a bunch of documents that sometimes don't load and causes the brat UI to become inaccessible (hover over the title bar doesn't do anything and text doesn't load). I managed to trigger this by typing in a single word into a document. My document looks like

watch

without any new lines or additional whitespace. "watch" seems to work. Any document that contains the phrase " watch  " fails to load.

**Comment by erintaylor** [comment]
I'm also having this issue, specifically in Firefox. The files with "watch" seem to load correctly in Chrome.

**Comment by bholland** [comment]
Yes. That is my experience as well. I didn't realize I was in firefox when
I wrote the initial bug. The files subsequently worked in Chrome.

Thanks,
~Ben
